src/mmrelay/meshtastic_utils.py

```
@@ -1454,21 +1454,32 @@ def _disconnect_sync(
+                    disconnect_method = client_obj.disconnect
+                    # Check _exit_handler on the client object safely
+                    client_exit_handler = getattr(client_obj, "_exit_handler", None)
+                    if client_exit_handler:
                         with contextlib.suppress(Exception):
-                            atexit.unregister(iface.client._exit_handler)
-                        iface.client._exit_handler = None
+                            atexit.unregister(client_exit_handler)
+                        try:
+                            client_obj._exit_handler = None
+                        except Exception:
+                            pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)